recipes/igraph/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "igraph" %}
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "d978030e27369bf698f3816ab70aa9141e9baf81c56cc4f55efbe5489b46b0df" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://igraph.org/nightly/get/c/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }} 
+
+build:
+  skip: True  # [ win ]
+  number: 3
+
+requirements:
+  build:
+    - gcc  # [not osx]
+    - gmp
+    - libxml2
+    - pkg-config
+  run:
+    - gmp 
+    - libgcc  # [not osx]
```

Comment:
Please drop this as it is unneeded. We supply a compiler and don't generally need to ship its libraries around.